### DISCUSSION TITLE
Special redirects do not appear to work

### DISCUSSION THREAD

**Discussion by ajf8729** [shlinkio/shlink#1596] [General]
#### How Shlink is set-up

* Shlink Version: 3.3.2
* PHP Version: 8.1.2-1
* How do you serve Shlink: Docker image (stable-roadrunner)
* Database engine used: SQLite

#### Summary

I have configured the special redirects with various URLs, but none of them seem to work. The bare URL always goes to a 404, and unknown shortlinks go to the Oops page. I am running it in Docker on an Ubuntu 22.04 VPS, with Apache serving as a reverse proxy to it, and SSL configured and redirecting HTTP -> HTTPS. I can manage it via the shlink app, and normally created shortlinks work fine. I really just want to get the bare URL to forward to my main blog/website.

Not sure what other info is needed or what can be done further to troubleshoot.

Thanks for making such a great and simple to use app!

-Anthony

**Comment by Bamstian** [comment]
Did you found a solution?
